sdk/ai/azure-ai-projects/samples/agents/sample_agents_fabric.py

```diff
@@ -0,0 +1,85 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_fabric.py
+
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the Microsoft Fabric grounding tool from
+    the Azure Agents service using a synchronous client.
+
+USAGE:
+    python sample_agents_fabric.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-identity
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+
+import os
+from azure.ai.projects import AIProjectClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import FabricTool
+
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
```

Comment:
The above 3 lines can be removed from all samples. I did that a while back for all Agent samples, but I see somehow it came back...